### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by swarnim-vyas**
This PR cleans up the Readme and updated description and commands

### CODE REVIEW

**Comment by mascharkh** [comment]
LGTM

**Review by mascharkh — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,37 +1,104 @@
...
+- Fifth parameter is an optional parameter. If --addDelay parameter is passed, delays are added randomly for the worker operations.
+
+7) Run driver for another terminal - 
+```
+cd Driver;
+python3 driver.py
+```
+Driver code runs all the supported operations in the system with predefined inputs.
+
+`Driver can be expanded to modify inputs ro run any supported operations `
```
Comment (aadhithNarayan) [inline README.md]:
```suggestion
`Driver can be expanded to modify inputs to run any supported operations `
```
Typo

File: README.md [hunk 2]
```diff
@@ -1,37 +1,104 @@
...
+```
+cd Scheduler;
+python3 scheduler_server.py --PortNumber 50051 --SchedulerMode PowerOf2
+```
+Reference Command:
+`python3 scheduler_server.py --PortNumber <PortNumber> --SchedulerMode <SchedulingMode> --AssingedWorkersPerTask <AssignedWorkersPerTask>`
+
+- First parameter is the port number. Type `int`.
+- Second parameter is the scheduling algorithm that would be used for request routing in the scheduler. Type `string`. It supports 4 modes `Random`, `RoundRobin`, `LoadAware` and `PowerOf2`
+- Third parameter is the number of workers a single task would assigned.  Type `int`. This Task Replication Strategy helps up retrieve a task computation from another worker when one of them is down.
```
Comment (aadhithNarayan) [inline README.md:59]: Please include 3rd param is optional, and it defaults to 1 which is no FT

Comment (krish8484) [reply README.md:59]: can be taken up in a subsequent PR if needs to be done

**Review by krish8484 — APPROVED** [review]

**Pull request merged**